### ISSUE TITLE
[bug] web tree sitter has different output than native tree-sitter

### ISSUE THREAD

**Issue by HerringtonDarkholme** [ast-grep/ast-grep#166]
Consider this code in Rust
```
Some($A)
```
On web tree sitter

<img width="721" alt="image" src="https://user-images.githubusercontent.com/2883231/211140703-f3963d97-1ac5-4472-a76a-c847b7c9bd03.png">

However, the output of `sg -p "Some($A)" -l rs --debug-query` is 

```
Pattern TreeSitter (call_expression function: (identifier) arguments: (arguments (identifier)))
```

Related issue:

https://github.com/tree-sitter/tree-sitter-rust/issues/82

**Comment by HerringtonDarkholme** [comment]
This can be fixed by #325

**Comment by HerringtonDarkholme** [comment]
won't fix because this is how tree-sitter works.
